### ISSUE TITLE
IoT LED integration?

### ISSUE THREAD

**Issue by dracan** [dracan/tomatoad#27]
Would be nice if there was an option to integrate into some form of coloured LED which you could put on your monitor so other people can see if you're currently in a pomodoro.

Doesn't have to be an LED - could even be something like a lava lamp. The integration should be quite abstracted so that we can support multiple things.

Needs research.

**Comment by Odonno** [comment]
You mean like an external device?

**Comment by dracan** [comment]
@Odonno Yep, just something that can be on your desk to let colleagues know you're not interruptable. How Tomatoad communicates with it depends on what devices are available (I need to do more research on this). So if there's an IoT light that is on the network with an IP address, and Tomatoad can set its colour based on your status - that kind of thing. I haven't looked into it yet - so it might not be feasible - I just created this issue to log the idea for later down the line.

**Comment by Odonno** [comment]
The best idea I get for now is to create a web server and use websockets to connect all your devices (phone, computers, IoT) so you can your Pomodoro info in realtime.

I know this idea is not a top priority so I'll leave as it is.
